pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -417,6 +417,12 @@ class Build {
 
         javaToBuild = javaToBuild.toUpperCase()
 
+	// Use version number for HEAD binaries
+	if (javaToBuild == "JDK") {
+		// This needs to be updated each time JDK-Head updates
+		javaToBuild = "15U"
```

Comment:
Should that be a lower case u?

Reply:
Line 418 sets `javaToBuild` as uppercase before this `if`